ReSharper.FSharp/test/data/parsing/Expr - Const - Measure - Divide 01.fs.gold

```diff
@@ -0,0 +1,26 @@
+﻿Language: PsiLanguageType:F#
+IFSharpImplFile
+  IAnonModuleDeclaration
+    IExpressionStatement
+      IChameleonExpression
+        ILiteralExpr
+          FSharpToken(type:INT32, text:123)
+          ITypeMeasure
+            FSharpTokenType+LessTokenElement(type:LESS, text:<)
+            IDivideMeasure
+              ISeqMeasure
```

Comment:
I think it would be easier to work with the tree if `seqMeasure` was only produced when there're more than one measure.